### PR TITLE
DlgMod fix show_message()

### PR DESCRIPTION
**Pull request by time-killer-games**

### CODE REVIEW

File: ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/** Copyright (C) 2018 Samuel Venable
+/** Copyright (C) 2019 Samuel Venable
```
Comment (JoshDreamland) [inline ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp]: Generally, leave the original year and use a hyphen (eg, 2018-2019).

Comment (RobertBColton) [reply ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp]: Yeah, I can't allow this. @time-killer-games you need to use a hyphen, you still own the copyright for 2018.

File: ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/** Copyright (C) 2019 Samuel Venable
+/** Copyright (C) 2018-2019 Samuel Venable
 ***
 *** This file is a part of the ENIGMA Development Environment.
 ***
```
Comment (RobertBColton) [inline ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp]: For future reference, if you make a commit you don't want built on the CI, you can put `[skip ci]` in your commit title. I will allow this for comments and simple changes where there is a previous commit that already passed the CI. This commit here is a perfect example of where I would have found it acceptable. Yes, this applies to commits made from GitHub UI as well as from git on the command line.

File: ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp (hunk 1 above)
Comment (RobertBColton) [reply ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp]: Ok, he fixed it in 3f39481c848855bf927c3c288ceb279baa9831eb

**Comment by RobertBColton** [comment]
I should mention that it's not clear what the purpose of this pull request was since it lacks a proper description. I can see that it looks like the removed external calls were not affecting the logic of the functions, which is a good reason to remove them. However, that leaves me asking what they were doing there in the first place?

**Comment by time-killer-games** [comment]
The were originally necessary. Changes have been made to the DLL and the function that was being called has been removed due to not being needed. Show_message when used would throw an error at compile time because I forgot to make it a const string.

File: ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp (hunk 2 above)
Comment (time-killer-games) [reply ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp]: Will do for comment edits here on out (btw in regards to me not "ever listening" to you it is not on purpose. I'm not as big a geek and my life gets wrapped up in other things in life so I often forget shit like this when not used often

**Comment by RobertBColton** [comment]
Thanks, that's fine, and for future reference it would be useful to just mention that in the pull request description. Then I would have been able to include that in the merge message. It's helpful to users so that they understand they need to get an updated copy of the DLL you are speaking of.

**Comment by time-killer-games** [comment]
Added it to description, here I go again with not listening to you. You told me like 5 things today each multiple times and I'm just now realising this

**Comment by RobertBColton** [comment]
That's fine, I didn't originally tell you to put it in the pull request description, or wasn't trying to. It's too late now since I already merged it. Please just remember for the future. I do make good points, and it's only going to make things easier for you to follow my advice.

**Pull request merged**